common/component_interface_specs/include/component_interface_specs/planning.hpp

```diff
@@ -39,6 +39,18 @@ struct SetRoute
   static constexpr char name[] = "/planning/mission_planning/set_route";
 };
 
+struct ChangeRoutePoints
+{
+  using Service = autoware_adapi_v1_msgs::srv::SetRoutePoints;
+  static constexpr char name[] = "/planning/mission_planning/change_route_points";
+};
+
+struct ChangeRoute
+{
+  using Service = autoware_planning_msgs::srv::SetRoute;
+  static constexpr char name[] = "/planning/mission_planning/change_route";
```

Comment:
@isamu-takagi cc @mitsudome-r @yukkysaito 
Could you please clarify why we need the `change` APIs in addition to the existing `set` APIs? If it's already documented, just writing the link is fine!

Reply:
Since the route change by MRM pull over is done while driving, it is necessary to keep the route for some distance. The `change route` interface diverts this so that it can be used in normal operations as well. Therefore, the interface is separated to call a different process than set route.

Reply:
@isamu-takagi Thank you, but when and why do users call this `change` API? :thinking:

Reply:
@kenji-miyake For example, when you want to change the goal while driving, or when splitting a route to create a circular route.

Reply:
Hmm, I'm sorry I don't understand the use cases well... :pleading_face: 
I'll leave the review to @mitsudome-r and @yukkysaito.